python/py-sentence-transformers/Portfile

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        UKPLab sentence-transformers 0.2.3 v
```

Comment:
This will have to be `0.2.4` after the impending release. See https://github.com/UKPLab/sentence-transformers/blob/v0.2.4/setup.py

Reply:
Updated now.